fbw-common/src/wasm/systems/systems/src/simulation/update_context.rs

```diff
@@ -314,6 +327,8 @@ impl UpdateContext {
             true_heading: Default::default(),
             plane_height_over_ground: Length::default(),
             latitude,
+            total_weight: Mass::new::<kilogram>(50000.),
```

Comment:
For consistency, I prefer this. (I sure hope the function `Mass::default` actually exists)

```suggestion
            total_weight: Mass::default(),
```